backend/plugins/frontline_api/src/modules/integrations/call/graphql/resolvers/customResolvers/customer.ts

```
@@ -0,0 +1,22 @@
+import { sendTRPCMessage } from 'erxes-api-shared/utils';
+
+export default {
+  async customer({ customerPhone }, {}, {}) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)